public/apps/account/log-out-button.tsx

```diff
@@ -16,20 +16,20 @@
 import React from 'react';
 import { EuiButtonEmpty } from '@elastic/eui';
 import { HttpStart } from 'opensearch-dashboards/public';
-import { logout } from './utils';
+import { logout, samlLogout } from './utils';
 
 export function LogoutButton(props: {
   authType: string;
   http: HttpStart;
   divider: JSX.Element;
   logoutUrl?: string;
 }) {
-  if (props.authType === 'openid' || props.authType === 'saml') {
+  if (props.authType === 'openid') {
```

Comment:
Does this bug affect oidc as well?

We recently discovered that session timeout also had the same bug where `setShouldShowTenantPopup(null)` was not getting called. See https://github.com/opensearch-project/security-dashboards-plugin/pull/1090

Reply:
Didn't test this for OIDC